config/manifests/bases/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Seamless Upgrades
+    olm.skipRange: '>=1.1.0 <1.8.0'
```

Comment:
is the plan for the first new operator be 1.8.0?
or if build as 2.0.0?

Reply:
The new operator version will be `2.0.0` , but the old operator releases will still continue. This just makes sure that any release of old operator is upgraded to new if they are on `alpha` or `fast` channel.

I will actually update it to be `<2.0.0`.  So it will just mean anything less than version  2.0.0 should upgrade to new operator in `fast` channel